INSTALL.rst

```diff
@@ -253,6 +253,9 @@ Dependencies for Language Bindings
   - `Cython <https://cython.org/>`_ >= 3.0.0
   - `scikit-build <https://pypi.org/project/scikit-build/>`_
   - `pytest <https://docs.pytest.org/en/6.2.x/>`_
+  - The source for `pythonic API <(https://github.com/cvc5/cvc5_pythonic_api)>`.
+    If configured with ``--pythonic-path=PATH``, the build system will expect that source to be at ``PATH``.
+    Otherwise, if configured with ``--auto-download``, the build system will download it.
 
```

Comment:
I'm not sure if we should actually put this here, since it is just another way to define the path to the pythonic API source code. Maybe add a note instead after this list.